app/org/sagebionetworks/bridge/config/BridgeSpringConfig.java

```diff
@@ -113,8 +117,8 @@ public void run() {
 
     @Bean(name = "healthCodeEncryptor")
     @Resource(name = "bridgeConfig")
-    public AesGcmEncryptor healthCodeEncryptor(BridgeConfig bridgeConfig) {
-        return new AesGcmEncryptor(bridgeConfig.getHealthCodeKey());
+    public BridgeAesGcmEncryptor healthCodeEncryptor(BridgeConfig bridgeConfig) {
```

Comment:
If we're going to be making use of interfaces, the return type should be BridgeEncryptor, not BridgeAesGcmEncryptor.